atlp-rwanda/c4-barefoot-frontend
# Ft authorised users' landing page

# Description

This is the branch for creating the page that a user lands on after logging in.


## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- checkout to ft-login-get-role on the backend.
- login with any user
- refresh the welcome page
- check the side menu; it is supposed to match the possible operation that the certain user can perform.

# Checklist:


# Screenshots (if applicable)
